camelot/admin/action/list_action.py

```diff
@@ -81,17 +82,25 @@ def get_state( self, model_context ):
         editable = model_context.field_attributes.get('editable', True)
         if editable == False:
             state.enabled = False
-        # Check for editability on the level of the entity
+        # Check if the action is authorized to perform this action's CRUD operation on the level of the entity
         admin = model_context.admin
-        if admin and not admin.is_editable():
-            state.visible = False
-            state.enabled = False
+        from vfinance.data.types import crud_types
+        if admin and admin.endpoint:
```

Comment:
moeten we hier noodzakelijk via de admin gaan ?  via de admin gaan maakt het mogelijk eenvoudiger de check te omzeilen